src/components/stockDiscoveryComponents/stockSearchResults/StockSearchResults.js

```diff
@@ -37,8 +37,9 @@ function StockSearchResults({keyword}) {
         {loading ? <LoadingSpinner /> : error  ? <MessageAlert variant='danger'>{error}</MessageAlert> :
         Object.keys(stocks).length !== 0  ?
         <>
+        <div className="stocksearchresults">
```

Comment:
Again have a look into Flexbox using containers, I see you have replaced Container with a div here and styling has effected the search bar. 

## Styling as Div 
When code is ```div className="stocksearchresults"```
![image](https://user-images.githubusercontent.com/61060096/204103519-fb2da12f-0d7a-49b2-a9ff-33fe25a6c155.png)

## Styling as Container 
When code is just ```<Container> ``` no class. You can use ```<div>``` if you want but just might be handier if bootsrap fixes these issues for you. 
![image](https://user-images.githubusercontent.com/61060096/204103653-89c4c657-7199-4167-9c39-b634f95d9d8c.png)